### PR TITLE
Script to convert data into bert-readable format

### PR DESCRIPTION
**Pull request by nthain**

### CODE REVIEW

File: experiments/tools/bert_tfrecord_converter.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+tf.app.flags.DEFINE_string('filenames', None,
+                           'Comma separated list of filenames.')
+tf.app.flags.DEFINE_string('bert_url', 'https://tfhub.dev/google/bert_uncased_L-12_H-768_A-12/1', 'TF Hub URL for BERT Model')
+tf.app.flags.DEFINE_integer('max_sequence_length', 256,
+                            'Maximum sequence length of tokenized comment.')
+
+FLAGS = tf.app.flags.FLAGS
+
+def create_int_feature(values):
+      f = tf.train.Feature(int64_list=tf.train.Int64List(value=list(values)))
```
Comment (iislucas) [inline experiments/tools/bert_tfrecord_converter.py]: Use 2 char intend consistently. :)

File: experiments/tools/bert_tfrecord_converter.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    in_filepath = '{}{}'.format(input_data_path, filename)
+    #TODO: Check if file exists, if not write new file
+    #TODO: Have the filename reflect the max_sequence_length and path reflect model
+    out_filepath = '{}{}'.format(output_data_path, filename)
+    record_iterator = tf.python_io.tf_record_iterator(path=in_filepath)
+    writer = tf.python_io.TFRecordWriter(out_filepath)
+    for ex_index, string_record in enumerate(record_iterator):
+      example = tf.train.Example()
+      example.ParseFromString(string_record)
+      text = example.features.feature[text_key].bytes_list.value[0]
```
Comment (iislucas) [inline experiments/tools/bert_tfrecord_converter.py]: Confused about where text_key and label_key are defined? Did you want this to be a flag?

File: experiments/tools/bert_tfrecord_converter.py (hunk 1 above)
Comment (nthain) [reply experiments/tools/bert_tfrecord_converter.py]: Thanks! Done.

File: experiments/tools/bert_tfrecord_converter.py (hunk 2 above)
Comment (nthain) [reply experiments/tools/bert_tfrecord_converter.py]: Ah, good catch! Yes, they should be flags and are now.

**Pull request merged**